npmrc

```diff
@@ -0,0 +1 @@
+registry=https://cli-npm.heroku.com
```

Comment:
I'm down with any host you folks prefer. Aesthetically npm is nicer than cli-npm.